lib/webpacker/test_helper.rb

```diff
@@ -0,0 +1,25 @@
+require "rake"
+require "webpacker"
+
+module Webpacker::TestHelper
+  extend ActiveSupport::Concern
+
+  included do
+    setup :compile_webpack_assets
+  end
+
+  def compile_webpack_assets
+    Rails.cache.fetch(["webpacker", "manifest", checksum]) do
```

Comment:
This cache key will only be invalidated when *your* files change, right? What if, for example, you bump the version of a dependency in package.json? I think it'd be best to avoid caching unless we can use the same digests provided by webpack.